src/core/features/google-analytics-instance-sharing.js

```diff
@@ -0,0 +1,24 @@
+const setupGoogleAnalyticsInstanceSharingFeature = (iframeRef, embedId) => {
+  // Throw an error if the feature is enabled but ga is not found
+
+  const gaObject = window[window.GoogleAnalyticsObject]
+
+  if (gaObject === undefined || gaObject === null) {
+    throw new Error(
```

Comment:
Should we use a try-catch block to better manage this function call? What happens when an error is thrown (E.g. shareGoogleAnalyticsInstance is set to true, but no gaObject is found)? Is the embedded form still being displayed?

Reply:
Thanks for getting back to me @supertinou 

IMHO Ideally, it should work as you said because it would be self-evident that something is wrong. 

However, to always display the form if we can, I will display an error in the console BUT continue with the execution and display the Typeform. 

For example, the GA code may be removed from the page because they want to use other analytics tools, then the form will stop working. Also, sometimes, non-technical people may be in charge of setting these GA codes (through Google Tag Manager, for example), they can accidentally remove the GA code, and then the form will stop working. 

So, to be on the safe side, I will display an error in the console but still show the form in these cases.

Reply:
Yes, we can keep the error in the console and the user will notice in their Google Analytics Dashboard if they have non matching users. Adding that to the integration for potential errors for the CS team!

Reply:
I addressed changes here: https://github.com/Typeform/embed/pull/108/commits/63b4b9847ed43ae5ea3b37a9dd7abd4477266a0a :)